src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -219,11 +219,20 @@ def create_chart_from_prometheus_query(
                                            'linejoin': 'round'},)
         plot_data_list.append(plot_data)
 
+    vertical_lines = []
+    horizontal_lines = []
     for line in lines:
```

Comment:
Why do we need this iteration?
No one is really using it, right?

Can we add the horizontal/vertical lines on the iteration on line 236?

Reply:
we already use this iteration for finding the max_y_axis which we later use in plotting the oom kill line.